### DISCUSSION TITLE
Module not found: Can't resolve '../../models/product' in '/vercel/path0/pages/product'

### DISCUSSION THREAD

**Discussion by IbrahimSSEnterprise** [vercel/community#791] [Help]
Hi Guys, 
I am getting this error wile deploying i have attached screen shot 
<img width="628" alt="Screenshot 2022-08-22 032346" src="https://user-images.githubusercontent.com/110059752/185875093-9d62d7cc-1f78-4ebb-9b90-b119bd2fcbc1.png">

**Comment by amyegan** [comment]
If you can share your repo or a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example), I would be happy to take a closer look!
